<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesAction</class>
 <widget class="QDialog" name="NotesAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Actions en masse</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Notes sélectionnées</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="selected_note_report">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="filter_combobox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Montrés</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cachés</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dont la note &gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dont la note &lt;</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Input" name="filter_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="3" colspan="2">
      <widget class="MultiNotesList" name="note_list">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="search_input">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Recherche</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="6">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="hide_button">
         <property name="text">
          <string>Cacher</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="show_button">
         <property name="text">
          <string>Montrer</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QPushButton" name="del_button">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Export</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="export_csv_button">
            <property name="text">
             <string>Export CSV</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Consommations</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1" colspan="2">
           <widget class="UniqueConsumptionList" name="product_list">
            <column>
             <property name="text">
              <string>Produits</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="take_products_btn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Valider</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QPushButton" name="refill_button">
            <property name="text">
             <string>Recharger</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="price_list">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QPushButton" name="take_button">
            <property name="text">
             <string>Prendre</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Input</class>
   <extends>QLineEdit</extends>
   <header>gui/input_widget</header>
  </customwidget>
  <customwidget>
   <class>MultiNotesList</class>
   <extends>QListWidget</extends>
   <header>gui/group_actions_window</header>
  </customwidget>
  <customwidget>
   <class>UniqueConsumptionList</class>
   <extends>QTreeWidget</extends>
   <header>gui/group_actions_window</header>
   <slots>
    <slot>on_selection()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>del_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>del_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hide_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>hide_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>show_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter_combobox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NotesAction</receiver>
   <slot>filter_combobox_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>export_csv_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>export_csv_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>take_products_btn</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>take_products_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>product_list</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>product_list</receiver>
   <slot>on_selection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refill_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>refill_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>take_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>take_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_input</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NotesAction</receiver>
   <slot>search_input_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>note_list</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>NotesAction</receiver>
   <slot>selection_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>del_action()</slot>
  <slot>hide_action()</slot>
  <slot>show_action()</slot>
  <slot>filter_combobox_change()</slot>
  <slot>filter_input_changed()</slot>
  <slot>export_xml_action()</slot>
  <slot>export_csv_action()</slot>
  <slot>take_products_action()</slot>
  <slot>refill_action()</slot>
  <slot>take_action()</slot>
  <slot>search_input_changed()</slot>
  <slot>selection_changed()</slot>
 </slots>
</ui>
